app/services/config_report_importer.rb

```diff
@@ -0,0 +1,9 @@
+class ConfigReportImporter < ReportImporter
```

Comment:
This class should take on some of the code from ReportImporter I think that is Puppet-specific, e.g. setting puppet_status, email notifications.